src/rdf4cpp/rdf/Literal.hpp

```diff
@@ -63,26 +64,41 @@ class Literal : public Node {
      */
     [[nodiscard]] datatypes::registry::DatatypeIDView datatype_id() const noexcept;
 
+    /**
+     * @return if the datatype of this is fixed
+     */
+    [[nodiscard]] bool is_fixed() const noexcept;
+
     /**
      * @return if the datatype of this is simultaneously fixed but not numeric
      */
     [[nodiscard]] bool is_fixed_not_numeric() const noexcept;
 
+
     /**
      * Creates a simple Literal directly without any safety checks
      */
     [[nodiscard]] static Literal make_simple_unchecked(std::string_view lexical_form, NodeStorage &node_storage) noexcept;
 
     /**
-     * Creates a typed Literal without doing any safety checks or canonicalization.
+     * Creates a non-inlined typed Literal without doing any safety checks or canonicalization.
      */
-    [[nodiscard]] static Literal make_typed_unchecked(std::string_view lexical_form, IRI const &datatype, NodeStorage &node_storage) noexcept;
+    [[nodiscard]] static Literal make_noninlined_typed_unchecked(std::string_view lexical_form, IRI const &datatype, NodeStorage &node_storage) noexcept;
 
     /**
      * Creates a language-tagged Literal directly without any safety checks
      */
     [[nodiscard]] static Literal make_lang_tagged_unchecked(std::string_view lexical_form, std::string_view lang, NodeStorage &node_storage) noexcept;
 
+    /**
+     * Creates an inlined Literal without any safety checks
+     */
+    [[nodiscard]] static Literal make_inlined_typed_unchecked(uint64_t inlined_value, storage::node::identifier::LiteralType fixed_id, NodeStorage &node_storage) noexcept;
```

Comment:
document parameter inlined_value. It should explain what's expected here because a uint64_t is used